src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  const coletion = [...list];
+
+  coletion.sort((a, b) => -(getIntegerNumber(a) - getIntegerNumber(b)));
```

Comment:
```suggestion
  coletion.sort((a, b) => getIntegerNumber(b) - getIntegerNumber(a));
```
it will be the same, but without additional braces